lib/inputs/file-pattern/static.ts

```diff
@@ -0,0 +1,64 @@
+import * as minimatch from "minimatch";
+import * as path from "path";
+
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToString } from "../../stream-utils";
+import { ManifestFile } from "../../types";
+
+function generatePathMatcher(
+  globsInclude: string[],
+  globsExclude: string[],
+): (filePath: string) => boolean {
+  return (filePath: string): boolean => {
+    let exclude = false;
```

Comment:
logic copied from the Docker class that already has it, consider extracting common things?

Reply:
deciding against it, we intend to delete the Docker code at some point so it would be a redundant abstraction